webrender/src/render_backend.rs

```diff
@@ -422,23 +444,22 @@ impl RenderBackend {
                 //           rebuild of the frame!
                 if let Some(property_bindings) = property_bindings {
                     doc.scene.properties.set_properties(property_bindings);
-                    doc.build_scene(&mut self.resource_cache);
+                    op.build = true;
```

Comment:
we should probably do the same for scroll & render events, instead of having `and_render` boolean